packages/comments/src/lib/comment-section.tsx

```diff
@@ -135,16 +122,20 @@ export function ButtonAndDialog({
 
 export function CommentSection({
   file,
```

Comment:
Remove all references to file (`FileLocation`,  `archive` and `filepath`) from all comments related code.

It should be replaces by `uri` and not `TOC+ slideNum`. This is general comments code for comments made outside slides too.